mlserver/parallel/worker.py

```diff
@@ -132,22 +150,38 @@ async def _process_model_update(self, update: ModelUpdateMessage):
                 "Unknown model update message with type ", update.update_type
             )
 
+    def _update_cb(self, update_task: Task):
+        err = update_task.exception()
+        if err:
+            logger.error(e)
+
+        self._model_updates.task_done()
+
+    def send_request(self, request_message: InferenceRequestMessage):
+        """
+        Send an inference request message to the worker.
+        Note that this method should be both multiprocess- and thread-safe.
+        """
+        loop = asyncio.get_event_loop()
```

Comment:
why do we need this statement?

Reply:
Your'e right, no need for this one - I think that was from copy & pasting the other method.